.ci/variables/variables_latest.yml

```diff
@@ -57,8 +57,8 @@ code_dockerfile_location: Dockerfile
 #. code_build_args (optional, dict):
 #+ Set build-time variables for the Docker build code_build_args:
 code_build_args:
-  PYTHON_VERSION: 2
-  ANSIBLE_VERSION: "2.4.*"
+  PYTHON_VERSION: 3
+  ANSIBLE_VERSION: "2.7.*"
```

Comment:
how should we manage the latest? Should it be the latest version builded ? Like 2.9

Reply:
Not sure, isn't 2.7 one of the popular ansible version out there, like most compatible ?

Reply:
This is a good question. Should the latest be the "popular" or the higher version we built.
I don't know how many customers still using "latest"

Reply:
latest will be ansible 2.9